pkg/storageincentives/agent.go

```diff
@@ -113,8 +113,10 @@ func New(overlay swarm.Address,
 
 	a.state = state
 
+	startWarmupPeriod := time.Now()
+
 	a.wg.Add(1)
-	go a.start(blockTime, a.blocksPerRound, blocksPerPhase)
+	go a.start(blockTime, a.blocksPerRound, blocksPerPhase, startWarmupPeriod)
```

Comment:
no need for the agent to know about the warmup period, let the function handle that